lib/fennec/udp.ex

```diff
@@ -14,6 +19,13 @@ defmodule Fennec.UDP do
   You can also start a server under Fennec's supervision tree
   using `start/1`.
 
+  ## Options
+
+  All methods of starting a server accepts the same configuration options
```

Comment:
All the methods...accept..passed as a keyword list:
...
... of an interface ... listen on